tictactoe.js

```diff
@@ -0,0 +1,136 @@
+
+player= "X";
+space = document.getElementsByClassName("space");
+
+
+// var boardCheck;
+
+// var cellValue = function cells(key) {
+//     switch(key) {
+//         case 'a': return "";
+//         case 'b': return "";
+//         case 'c': return "";
+//         case 'd': return "";
+//         case 'e': return "";
+//         case 'f': return "";
+//         case 'g': return "";
+//         case 'h': return "";
+//         case 'i': return "";
+//         default : return "";
+//     }
+// }
+
+
+//-------------------------------------------------------------------
+function clickSpace() {
+    if (this.id == "a") {
+        if(document.getElementById("a").innerHTML == ""){ 
+           document.getElementById("a").innerHTML == player;
```

Comment:
This line here is giving you a bug that causes the first square not to change. It should be the assignment operator instead of the comparison operator.

```
document.getElementById("a").innerHTML = player;
```